mapogr.cpp

```diff
@@ -2353,6 +2293,78 @@ static int msOGRFileWhichShapes(layerObj *layer, rectObj rect, msOGRFileInfo *ps
                       EQUAL(psInfo->dialect, "GPKG")) &&
                      psInfo->pszMainTableName != NULL )
             {
+                if( psInfo->dialect && psInfo->pszMainTableName != NULL && 
```

Comment:
this first line of test in useless as they have done just above

Reply:
yup, you are correct